src/index.jsx

```diff
@@ -2,7 +2,6 @@ import 'babel-polyfill';
 import React from 'react';
 import ReactDOM from 'react-dom';
 import {
-  identifyAnonymousUser,
```

Comment:
@robrap indicated we shouldn't be publishing an identifyAnonymousUser event if we have no accessToken.  So I've removed it and where it was called below.